src/backend/distributed/commands/extension.c

```diff
@@ -626,7 +626,7 @@ MarkExistingObjectDependenciesDistributedIfSupported()
 	ObjectAddress *objectAddress = NULL;
 	foreach_ptr(objectAddress, uniqueObjectAddresses)
 	{
-		MarkObjectDistributed(objectAddress);
+		MarkObjectDistributed(objectAddress, "");
```

Comment:
instead of adjusting the existing calls, probably nicer to make 2 variants

(either 2 separate variants of MarkObjectDistributed, or make MarkObjectDistributed a wrapper around an extended variant)